articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,103 @@
+
+Once this is all set up, when the user attempts to authenticate as normal, their device will receive a push notification via the app, and once they approve the request, they will be logged in.
+
+Auth0 supports the use of Auth0 Guardian ([Google Play](https://play.google.com/store/apps/details?id=com.auth0.guardian) / [App Store](https://itunes.apple.com/us/app/auth0-guardian/id1093447833?mt=8)) and Duo ([Google Play](https://play.google.com/store/apps/details?id=com.duosecurity.duomobile) / [App Store](https://itunes.apple.com/us/app/duo-mobile/id422663827?mt=8)) for push notifications.
+
+::: note
+Enabling Duo will result in end users not being given any other choices for MFA factors, even if other factors are enabled and set up.
+:::
+
+#### SMS
+
+SMS as an MFA factor means that the end user is sent a code via SMS when attempting to authenticate with your application. They will have to enter this code to complete the transaction, implying that in addition to knowing their login information, they also have posession of the device set up for receiving MFA texts.
+
+Your users must have a device capable of using SMS to use this option. If your users are unable to always receive SMS messages (such as when traveling), they will be unable to sign up with SMS and will be unable to log in without SMS or their recovery code.
+
+After signing up, and entering a country code and phone number, the user will receive a six digit code to their device. They need to enter this code into the box, and then they will also get a recovery code.
+
+In order to set up SMS, you will need to [configure an SMS provider such as Twilio](/multifactor-authentication/twilio-configuration). You can also [customize your SMS notification templates](/multifactor-authentication/sms-templates).
+
+## Customizing multi-factor authentication
+
+### Customizing via Rules
+
+If you need to customize the multi-factor experience you are offering to your users, you may do so via [custom rules configurations for multi-factor authentication](/multifactor-authentication/custom). This might be needed, for example, if you wish to defined conditions (such as changes in geographic location or logins from unrecognized devices) which would trigger additional authentication challenges.
+
+### Customizing Guardian
+
+The hosted page for Guardian can also be customized. You may change the logo and the name that is displayed to your users. To do so, make the appropriate changes to the Guardian page's settings on the **General** tab in [Dashboard > Tenant Settings](${manage_url}/#/tenant). You can also reach the Tenant Settings page by clicking on your tenant name on the top right of the page and then selecting **Settings** from the dropdown menu.
```

Comment:
Should we say 'hosted page'? We are moving to 'Universal Login' mostly everywhere I think

Reply:
Yeah, when referencing the login page itself we as a rule say Universal Login or, where it makes sense contextually, just "login page". For hosted error pages or the hosted Guardian page, I am unsure how we'd change the terminology. I could drop the word "hosted" if you'd like, except in Dashboard instructions since that's still the name there ("Hosted Pages").